ACPs/77-reinventing-subnets/README.md

```diff
@@ -109,21 +76,77 @@ type SubnetValidator struct {
     Validators []SubnetValidator `json:"validators"`
-    // Auth that will be allowing these validators into the network
+    // Authorizes this conversion
     SubnetAuth verify.Verifiable `json:"subnetAuthorization"`
 }
 ```
 
+Once this transaction is accepted, `AddSubnetValidatorTx` is disabled on the Subnet. The only action that the `Owner` key is able to take is removing previously existing Subnet validators via `RemoveSubnetValidatorTx`. Any previously existing validators added with the pre-ACP-77 workflow will otherwise be removed automatically at their specified [`End`](https://github.com/ava-labs/avalanchego/blob/a1721541754f8ee23502b456af86fea8c766352a/vms/platformvm/txs/validator.go#L27) time. `RegisterSubnetValidatorTx` and `SetSubnetValidatorWeightTx` must be used to manage the Subnet's validator set going forward, and the `Owner` key is powerless to modify anything related to these newly added validators.
+
+The `validationID` for validators added in `ConvertSubnetTx` is defined as the SHA256 hash of `(convertSubnetTxID,validatorIndex)`, where `validatorIndex` refers to the index into the `Validators` array of within the `ConvertSubnetTx`.
+
+The following serialization is defined as an `InitialSubnetValidator`:
+```text
++--------------+----------+-----------+
+|       nodeID : [32]byte |  32 bytes |
++--------------+----------+-----------+
+|       weight :   uint64 |   8 bytes |
++--------------+----------+-----------+
+| blsPublicKey : [48]byte |  48 bytes |
++--------------+----------+-----------+
+                          |  88 bytes |
+                          +-----------+
+```
+The following serialization is defined as the `SubnetConversionData`
+```text
++-------------------+--------------------------+--------------------------------------------------------------+
+| convertSubnetTxID :                 [32]byte |                                                     32 bytes |
++-------------------+--------------------------+--------------------------------------------------------------+
+|    managerChainID :                 [32]byte |                                                     32 bytes |
++-------------------+--------------------------+--------------------------------------------------------------+
+|    managerAddress :                   []byte |                                4 + len(managerAddress) bytes |
++-------------------+--------------------------+--------------------------------------------------------------+
+| initialValidators : []InitialSubnetValidator |                        4 + len(initialValidators) * 88 bytes |
++-------------------+--------------------------+--------------------------------------------------------------+
+                                               | 72 + len(managerAddress) + len(initialValidators) * 88 bytes |
+                                               +--------------------------------------------------------------+
```

Comment:
`initialValidators` feels a little misleading since the Subnet can have other validators from `AddSubnetValidatorTx`. I think dropping initial would be best since these are strictly the validators added in the conversion tx.

Reply:
Updated 👍